recipes/micromamba/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.3.8" %}
+
+package:
+  name: micromamba
+  version: {{ version }}
+
+source:
+  url: https://github.com/TheSnakePit/mamba/archive/{{ version }}.tar.gz
+  sha256: 2c1029d350be219c682ea803004cadd4dc40c9a5029784769684530c9a17c0ef
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+  host:
+    - cli11
+    - nlohmann_json
+    # static libs
+    - libcurl
+    - libarchive
+    - krb5-static
+    - libsolv-static
+    - libopenssl-static
+
+test:
+  commands:
+    - test -f $PREFIX/bin/micromamba  # [unix]
+
+about:
+  home: https://github.com/TheSnakePit/mamba
+  license_file: LICENSE
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
This isn't needed any more for SPDX licences identifiers:
```suggestion
```

Reply:
It is OK to keep it in though